### PR TITLE
ZF3 compatibility (No BC break)

### PR DESCRIPTION
**Pull request by shandyDev**
Hi guys!
I implemented todo to support ZF3 in this pull request. Also was
- set minimum supported PHP version to 5.6 (required by ZF)
- code style (using ::class)
- update composer.json to tagged latest version of dependencies
- Exposed to zend-component-installer as ZF module in composer.json

Please check & review this. If all ok - I suggest create v1.0 release with it (or 0.5.0?).

### CODE REVIEW

**Comment by shandyDev** [comment]
BTW, tests was passed.

**Comment by shandyDev** [comment]
Ping @ojhaujjwal

**Comment by ojhaujjwal** [comment]
@shandyDev Thanks a lot man. I really appreciate your effort. I will review the PR in 1 or 2 days.

**Comment by ojhaujjwal** [comment]
first look and it looks absolutely awesome. I am really glad to see your contribution.

File: composer.json [hunk 1]
```diff
@@ -16,31 +17,39 @@
...
-        "zendframework/zend-modulemanager": "~2.2",
-        "zendframework/zend-stdlib": "~2.2",
-        "zendframework/zend-http": "~2.2",
-        "zendframework/zend-eventmanager": "~2.2",
+        "php": "^5.6 || ^7.0",
+        "zendframework/zend-mvc": "^2.7.10 || ^3.0",
+        "zendframework/zend-servicemanager": "^2.7.6 || ^3.1",
+        "zendframework/zend-modulemanager": "^2.6",
+        "zendframework/zend-stdlib": "^2.7 || ^3.0",
+        "zendframework/zend-hydrator": "^1.1 || ^2.2",
```
Comment (shandyDev) [inline composer.json]: You are right, this is overhead. Fixed.

**Comment by ojhaujjwal** [comment]
@shandyDev I don't use Zend Framework these days. So, I don't really use this library to be honest. So, this library is currently unmaintained. But, I will be always happy to accept PRs if they are good enough. If you are interested in becoming a maintainer, I will be happy to add you as a maintainer of this repo. You can create PRs and I will review them and you can review PRs that others open.

**Comment by shandyDev** [comment]
Unexpected, but pleasant proposal. )
So, I used your module on couple projects with ZF, which is why we need ZF3 support.
In general, I will be glad to become a maintainer.
And by the way, I have some other suggestions which we implemented in our projects.

**Comment by ojhaujjwal** [comment]
looks good to me this PR. if you are done with it, I will merge the PR and add you as maintainer of this library.

**Comment by shandyDev** [comment]
Yes, I tested in real project and it seems all works as expected. So you can merge it.

**Comment by shandyDev** [comment]
@ojhaujjwal Big thanks!
Also you are right with 0.5.0 tag, not nessesary create 1.0.0 release. Maybe in future, when implemented for example CacheResolver or integration with AssetManager module...

BTW: Last PR (https://github.com/hrevert/HtImgModule/pull/27) this is also was employee @MaxVoronov from our company. So we already use this module long time. )

**Pull request merged**